src/cockpit/data/fail.html

```diff
@@ -0,0 +1 @@
+../../../src/common/fail.html
```

Comment:
I hope that pybuild, pip, beiboot, and whatnot get along with symlinks. Otherwise it would really not be a big deal to copy the file, the one in src/common will disappear at some point, right?